serveroptions.go

```diff
@@ -54,10 +57,24 @@ func WithHTTPReadTimeout(hctime time.Duration, wstime time.Duration) ServerOptio
 	})
 }
 
-// Creds returns a ServerOption that sets credentials for server connections.
-func Creds(privKey ed25519.PrivateKey, pubKeys []ed25519.PublicKey) ServerOption {
+func WithWSReadLimit(numBytes int64) ServerOption {
```

Comment:
Does the `WS` add necessary context? Or could it be dropped?
```suggestion
func WithReadLimit(numBytes int64) ServerOption {
```

Reply:
Eh kinda looks like it matches local style already.